### DISCUSSION TITLE
Error only log file

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#119] [Q&A] [answered]
Hello

Is there a way to only have error messages (and maybe warnings) in a specific log file, in addition to the general log file that can be created that shows all the steps of the processes). At this moment i output everything onto a general logfile and then use for example notepad pad to find errors (having used "ignore errors" in the script when i run it)

Txs for the help  

Cheers
Jean

**Comment by tmayerguerr** [comment, marked as answer]
Hi Jean,
This is not planned. Maybe you can redirect the error messages to an extra file: See for example https://learn.microsoft.com/en-us/troubleshoot/developer/visualstudio/cpp/language-compilers/redirecting-error-command-prompt
regards
Torsten

**Comment by Bemtgenj** [comment]
Hi Torsten

txs for the feedback - i will take a look at your suggestion

cheers
Jean
